int32[32] data
int8 slave_id
int8 func_code

